topic-controller/src/main/java/io/strimzi/controller/topic/Session.java

```diff
@@ -166,9 +166,14 @@ private void reconcileTopics(String reconciliationType) {
                     TopicName topicName = new TopicName(name);
                     k8s.getFromName(topicName.asMapName(), ar -> {
                         ConfigMap cm = ar.result();
-                        // TODO need to check inflight
-                        // TODO And need to prevent pileup of inflight periodic reconciliations
-                        controller.reconcile(cm, topicName);
+
+                        controller.reconcile(cm, topicName, reconcileResult -> {
+                            if (reconcileResult.succeeded()) {
+                                LOGGER.info("Success reconciling topic {}", topicName);
```

Comment:
You could also add the `reconciliationType` to these log messages, for extra context. Also, is there a reason you're not logging the `cm` name here?

Reply:
No reasons I'm going to add both.